TOOLS/lua/autocrop.lua

```diff
@@ -26,55 +28,69 @@
 -- to mpv's arguments. This may be desirable to allow cropdetect more
 -- time to collect data.
 require "mp.msg"
+require 'mp.options'
 
 script_name = mp.get_script_name()
 cropdetect_label = string.format("%s-cropdetect", script_name)
 crop_label = string.format("%s-crop", script_name)
 
--- number of seconds to gather cropdetect data
-detect_seconds = tonumber(mp.get_opt(string.format("%s.detect_seconds", script_name)))
-if not detect_seconds then
+local options = {
+    auto = "no",
```

Comment:
mpv.conf give string like "yes", false doesn't work

Reply:
The the [read options](https://mpv.io/manual/master/#lua-scripting-on-update]]\)) entry in the manual has an example that uses boolean. This will translate to yes / no for user on the command line or conf file. The reason it's not working for you is probably because you are doing `options.auto == "yes"` rather than just `options.auto`